docs/src/user-guide/core-clp-s.md

```diff
@@ -1,14 +1,7 @@
-# Using CLP for semi-structured logs
+# clp for dynamically-structured logs
```

Comment:
Is lower case clp correct? I think most other places in these docs use upper case CLP.

Reply:
Usually I differentiate CLP the brand from clp the core component, but I agree it's confusing in this instance. I switched the "CLP for xxx" headings to uppercase.